_posts/2023-11-16-quarkus-meets-langchain4j.adoc

```diff
@@ -0,0 +1,236 @@
+---
+layout: post
+title: 'When Quarkus meets LangChain4j'
+date: 2023-11-15
+tags: AI langchain
+synopsis: 'Learn about the new quarkus-langchain4j extension to integrate LLMs in Quarkus applications.'
+author: cescoffier
+---
+:imagesdir: /assets/images/posts/llm
+
+Large language models (LLMs) are reshaping the world of software, altering the way we interact with users and develop business logic.
+
+Populatized by https://openai.com/[OpenAI]'s https://chat.openai.com/[ChatGPT], LLMs are now available in many flavors and sizes. The https://huggingface.co/models[Hugging-Face] platform references hundreds of them, and major tech companies like Facebook, Google, Microsoft, and Amazon are also providing their own models.
+
+LLMs are not a new concept. They have been around for a while, but they were not as powerful as they are today. The Quarkus team have been thinking about what it would mean to integrate LLMs in Quarkus applications. The talk https://www.youtube.com/watch?app=desktop&v=BD1MSLbs9KE[Java Meets AI] from Lize Raes at Devoxx 2023 has been a great source of inspiration.
```

Comment:
```suggestion
LLMs are not a new concept. They have been around for a while, but they were not as powerful or as accessible the became when OpenAI made ChatGPT API's publically available. Since then the Quarkus team have been thinking about what it would mean to integrate LLMs in the Quarkus ecosystem. The talk https://www.youtube.com/watch?app=desktop&v=BD1MSLbs9KE[Java Meets AI] from Lize Raes at Devoxx 2023 has been a great source of inspiration.
```